src/Microsoft.Identity.Client/TokenCacheExtensions.cs

```diff
@@ -43,48 +44,53 @@ namespace Microsoft.Identity.Client
     /// format, and the new unified cache format, common to ADAL.NET, MSAL.NET, and other libraries on the same platform (MSAL.objc, on iOS)</remarks>
     public static class TokenCacheExtensions
     {
+        private const string AccessTokenKey = "access_tokens";
+        private const string RefreshTokenKey = "refresh_tokens";
+        private const string IdTokenKey = "id_tokens";
+        private const string AccountKey = "accounts";
+
         /// <summary>
         /// Sets a delegate to be notified before any library method accesses the cache. This gives an option to the
         /// delegate to deserialize a cache entry for the application and accounts specified in the <see cref="TokenCacheNotificationArgs"/>.
         /// See https://aka.ms/msal-net-token-cache-serialization
         /// </summary>
-        /// <param name="tokencache">Token cache that will be accessed</param>
+        /// <param name="tokenCache">Token cache that will be accessed</param>
         /// <param name="beforeAccess">Delegate set in order to handle the cache deserialiation</param>
         /// <remarks>In the case where the delegate is used to deserialize the cache, it might
         /// want to call <see cref="Deserialize(TokenCache, byte[])"/></remarks>
-        public static void SetBeforeAccess(this TokenCache tokencache, TokenCache.TokenCacheNotification beforeAccess)
+        public static void SetBeforeAccess(this TokenCache tokenCache, TokenCache.TokenCacheNotification beforeAccess)
```

Comment:
>tokenCache [](start = 59, length = 10)

while not really breaking, I wonder if this will cause xcopy deployments to fail?

Reply:
It shouldn't since the param names aren't actually mapped in the final assembly as something contractual.

Reply:
We're also not supporting xcopy deployments since this is a public API breaking change release.

---
In reply to: [241282407](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/723#discussion_r241282407) [](ancestors = 241282407)